### DISCUSSION TITLE
Seattle Area Hospitals Locator

### DISCUSSION THREAD

**Discussion by mhan799** [jakobzhao/geog495#60] [Final Project]
Link: https://mhan799.github.io/seattle_hospitals/index.html
GitHub: https://github.com/mhan799/seattle_hospitals

Desc: This application is intended as a guide and is based on the historical data of hospitals from the Seattle GeoData. This guide included basic information of facility name, address, and phone number in order to let people get an image of the hospitals' distributions of the Seattle area and to get help if non-emergency cases occur. If an emergency happens, we recommend calling the ambulance at once. 
<img width="1277" alt="Screen Shot 2021-12-09 at 9 55 10 PM" src="https://user-images.githubusercontent.com/49173975/145527263-9a8e381f-b69e-4b00-8528-e0ed00246609.png">

**Comment by zeyany2** [comment]
Hi! Thank you for your great Job! Your idea is great and this application is simple for users to understand. Some suggestions I could give are: making a search bar and can find the neareast hospital for a certain location. Changing the base map to street view so that users can have a better understanding about the area.

**Comment by aembar** [comment]
Really nice job! I like the layout of your map. It's really intuitive and the colors are all consistent in the application. Some things I would suggest are: removing the gap in the popup between the address and number, and adding in a close button on the popup so that you can reset the application back to its original state.

**Comment by alexanderlyu** [comment]
I like the amount of information displayed on the sidebar. It contains all the necessary information that really highlights the usage of your application. However, I think it would be better if you could move the phone number to its own line, as the addresses of some of the hospitals push the phone number to the next line and creates an awkward break. It would be much clearer and easier to understand if the phone number was consistently on its own.

**Comment by ghost** [comment]
I really like the style choice with this type of information map. As hospitals are emergency use, it is better visual hierarchy to keep them visible and other features dulled out for quick use, like what you did in the screenshot. However, when I clicked on your link, there was a much more colorful theme used, I think you should try a duller scheme personally!

**Comment by dldl1901s** [comment]
Your map design is amazing. The purpose of the map is very clear, and there is no part where the map does not functionally work. Personally, it is particularly good to have a green design as a whole.

**Reply by whalen323** [reply]
My project was very similar! I like the simplicity and how easy to use and understand it is, with the nice light background! One suggestion that I have is to include a location finder, that way people could see what the nearest hospital is from anywhere they are!
